src/main/resources/site/i18n/phrases.properties

```diff
@@ -11,6 +11,7 @@ modified = Endret
 notAvailableHeader = This page is not available in English
 notAvailableBody = For content in English,<br/>please visit our homepage
 goHome = Go to home
+close = Close
```

Comment:
The correct phrase here would be "Lukk". This is the default phrases file!

Reply:
Oi, retta feil fil. Whoopsie!